assignments/prototypes.js

```diff
@@ -32,7 +54,20 @@
   * should inherit destroy() from GameObject through CharacterStats
   * should inherit takeDamage() from CharacterStats
 */
- 
+function Humanoid(humanStats) {
+  this.team = humanStats.team;
+  this.weapons = humanStats.weapons;
+  this.language = humanStats.language;
+  
+  GameObject.call(this, humanStats);
```

Comment:
Since CharacterStats is already inheriting from GameObject it is not necessary to call GameObject again. However, it will not have a negative effect on your code.